### PR TITLE
feat: add custom user-agent to queries

### LINKED ISSUE #776: Add custom User-Agent to app's axios calls
I have my own node I run ReStake against, and when analyzing node's logs, I see entries like these in nginx logs:

```
<source-ip> - - [02/May/2024:01:19:35 +0200] "GET /cosmos/distribution/v1beta1/delegators/<address>/rewards HTTP/1.1" 200 335 "-" "axios/1.3.4"
```

I know that it's ReStake doing these, but having ReStake somehow identifying itself and its headers via User-Agent header would be nice, to differentiate these from other queries, as if some other JS-based tool using axios as a library to fetch data from would also query my node, it would be impossible to distinguish the two.

(Feel free to assign me if you want, I can make a PR adding it.)

### PR DESCRIPTION
**Pull request by freak12techno**
Fixes #776.

Moved get() and post() of axios to helpers to set a custom User-Agent header.

### CODE REVIEW

File: src/utils/constants.mjs [hunk 1]
```diff
@@ -0,0 +1 @@
+export const RESTAKE_USER_AGENT = 'ReStake'
```
Comment (tombeynon) [inline src/utils/constants.mjs:1]:
```suggestion
export const RESTAKE_USER_AGENT = 'REStake'
```

**Review by tombeynon — COMMENTED** [review]
This is awesome, thanks so much for the clean solution. I'll give it a test and we can get it merged asap. One minor change to the header value

File: src/utils/constants.mjs (hunk 1 above)
Comment (freak12techno) [reply src/utils/constants.mjs]: @tombeynon I also thought it'd be nice to include the version/commit or something, but restake doesn't have versioning, right?

Comment (tombeynon) [reply src/utils/constants.mjs]: I was thinking the same, sadly not. The Git versions are good but the app doesn't know about them. I'll raise an issue to get that added.

Comment (freak12techno) [reply src/utils/constants.mjs]: Node's package.json is also nice, if doing it via package.json version we can just import it as json file, so getting the version would be pretty straightforward.
Also suggest looking into libraries that are making GitHub releases and stuff, I use goreleaser for all of the Golang tools, guess there should be something similar in JS ecosystem. One solution I found by briefly googling is semantic-release.

**Pull request merged**